about/policy/data-sharing-agreement/index.md

```diff
@@ -0,0 +1,69 @@
+---
+title: State Geographic Information Database (SGID) Data Sharing Agreement
+layout: page_nocrumb
+author:
+  display_name: UGRC Office
+  email: ugrc@utah.gov
+tags:
+  - license
+  - policy
+categories: []
+date: 2020-01-25 17:00:00 -0700
+---
+
+The following Data Sharing Agreement applies to all individuals and organizations sharing data with the SGID in any way.
+
+`<Four different levels of getting agreement/acknowledgement, from easiest to most onerous>`
+
+In order to ~~emulate the automatic EULAs of big, powerful software companies~~ reduce the friction of adding data to the SGID, by completing a [Porter issue](https://github.com/agrc/porter/issues) to submit your data to the SGID, you agree to this Data Sharing Agreement.
```

Comment:
i think the joke could be removed.

Reply:
Party pooper :) Yeah, if/when we settle on this method (sounds like we're close, but I'd like to get feedback from @brigsz and @gregbunce) I'll clean it up.